frontend/admin/src/js/components/poolCandidate/PoolCandidatesTable.tsx

```diff
@@ -1,217 +1,478 @@
+type Data = NonNullable<FromArray<PoolCandidatePaginator["data"]>>;
 
 // callbacks extracted to separate function to stabilize memoized component
-const languageAbilityAccessor = (
-  languageAbility: LanguageAbility | null | undefined,
+const preferredLanguageAccessor = (
+  language: Language | null | undefined,
   intl: IntlShape,
 ) => (
   <span>
-    {languageAbility
-      ? intl.formatMessage(getLanguageAbility(languageAbility as string))
-      : ""}
+    {language ? intl.formatMessage(getLanguage(language as string)) : ""}
   </span>
 );
-const preferredLanguageAccessor = (
-  language: Language | null | undefined,
+const statusAccessor = (
+  status: PoolCandidateStatus | null | undefined,
+  intl: IntlShape,
+) => {
+  if (status === PoolCandidateStatus.NewApplication) {
+    return (
+      <span data-h2-color="base(dt-accent)" data-h2-font-weight="base(700)">
+        {status
+          ? intl.formatMessage(getPoolCandidateStatus(status as string))
+          : ""}
+      </span>
+    );
+  }
+  if (
+    status === PoolCandidateStatus.ApplicationReview ||
+    status === PoolCandidateStatus.ScreenedIn ||
+    status === PoolCandidateStatus.ScreenedOutApplication ||
+    status === PoolCandidateStatus.UnderAssessment ||
+    status === PoolCandidateStatus.ScreenedOutAssessment
+  ) {
+    return (
+      <span data-h2-font-weight="base(700)">
+        {status
+          ? intl.formatMessage(getPoolCandidateStatus(status as string))
+          : ""}
+      </span>
+    );
+  }
+  return (
+    <span>
+      {status
+        ? intl.formatMessage(getPoolCandidateStatus(status as string))
+        : ""}
+    </span>
+  );
+};
+const priorityAccessor = (
+  priority: number | null | undefined,
+  intl: IntlShape,
+) => {
+  switch (priority) {
+    case 10:
+      return (
+        <span data-h2-color="base(dt-primary)" data-h2-font-weight="base(700)">
+          {intl.formatMessage({
+            defaultMessage: "Priority Entitlement",
+            id: "j1p7LR",
+            description: "Priority text for users with priority entitlement",
+          })}
+        </span>
+      );
+    case 20:
+      return (
+        <span data-h2-color="base(dt-primary)" data-h2-font-weight="base(700)">
+          {intl.formatMessage({
+            defaultMessage: "Veteran",
+            id: "oU8C65",
+            description: "Priority text for veterans",
+          })}
+        </span>
+      );
+    case 30:
+      return (
+        <span>
+          {intl.formatMessage({
+            defaultMessage: "Citizen or Resident",
+            id: "oMyc4e",
+            description: "Priority text for citizens of canada",
+          })}
+        </span>
+      );
+    default:
+      return (
+        <span>
+          {intl.formatMessage({
+            defaultMessage: "Work Visa",
+            id: "EimWiB",
+            description: "Priority text for users with work visas",
+          })}
+        </span>
+      );
+  }
```

Comment:
Hmm, if we plan on using this elsewhere could we move this to `localizedConstants.tsx` with something like:

```tsx
export const poolCandidatePriorities = defineMessages({
  [10]: {
    defaultMessage: "Priority Entitlement",
    ...
  },
  ...
});

```

Reply:
Added with [Moved status text to localized constants and changed 'Work Visa' to 'Other'](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3956/commits/4bddfcc5e268de11913d07daf3ec5aca523ed483)